src/main/native/smithwaterman/PairWiseSW.h

```diff
@@ -1,4 +1,5 @@
 #include<stdio.h>
+#include<limits.h>
```

Comment:
Why do you include limits.h if you don't use any new functions?

Reply:
I wanted to check, if we need any check to replace upper-bound here e.g.`if (curSize < (maxSize - 4)) `